### PR TITLE
Fixed Lua cheat sheet: Changed else if to elseif (#2071)

### LINKED ISSUE #2071: Lua Language Cheat Sheet: Incorrect else if Statement
@lxndio 
The else if statement in Lua is noted as `elseif`, not `else if`.

------
IA Page:  http://duck.co/ia/view/lua_cheat_sheet

### PR DESCRIPTION
**Pull request by lxndio**
Changed `else if` to `elseif`.
Closes #2071

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/lua.json [hunk 1]
```diff
@@ -49,6 +49,138 @@
         "val": "Or"
       }
     ],
+<<<<<<< HEAD
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/lua.json]: remove this to get checks passing

File: share/goodie/cheat_sheets/json/lua.json [hunk 2]
```diff
@@ -49,6 +49,138 @@
...
+                "val": "For loop with one step for each entry (value) in a table. 'key' is either the index or the key when using custom keys."
+            },
+            {
+                "key": "while true do", 
+                "val": "While loop"
+            }
+        ]
+    }
+}
+=======
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/lua.json]: also this

**Comment by jagtalon** [comment]
:+1: nice!

**Pull request merged**